dbt/include/global_project/macros/materializations/snapshots/strategies.sql

```diff
@@ -70,7 +70,16 @@
         ({{ snapshotted_rel }}.{{ columns.dbt_valid_from }} < {{ current_rel }}.{{ updated_at }})
     {%- endset %}
 
-    {% set scd_id_expr = snapshot_hash_arguments([primary_key, updated_at]) %}
+    {% if primary_key is iterable and primary_key is not string and primary_key is not mapping %}
```

Comment:
I would make this its own macro as well. Are `iterable` and `sequence` different in jinja? Could we check both in a shared macro if they are?

Reply:
This just wasn't updated to use the new 'is_list' filter that I added in dbt-common.